src/app/app-platform/ContentApp.h

```diff
@@ -47,6 +48,8 @@ using KeypadInputDelegate         = app::Clusters::KeypadInput::Delegate;
 using MediaPlaybackDelegate       = app::Clusters::MediaPlayback::Delegate;
 using TargetNavigatorDelegate     = app::Clusters::TargetNavigator::Delegate;
 
+using chip::Protocols::InteractionModel::Status;
```

Comment:
No `using` in headers, please.  Stopping here; there will be a bunch of changes to fix this, I expect.